src/scripts/main.js

```diff
@@ -353,8 +357,30 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+const table = document.querySelector('.dashboard');
+
+function addRowToTable(person) {
+  const row = document.createElement('tr');
+
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
```

Comment:
The calculation of the century using `Math.ceil(person.died / 100)` is correct for determining the century based on the year of death. Ensure that this logic aligns with the task requirements, as it assumes the century is determined by the year of death.